canopen_monitor/parse/pdo.py

```diff
@@ -108,7 +108,7 @@ def parse_pdo(num_elements, pdo_type, cob_id, eds, data):
                                         f"Unable to find eds data for pdo type"
                                         f" {hex(pdo_type)} index {i}")
 
-        pdo_definition = int(eds_record.default_value, 16).to_bytes(4, "big")
+        pdo_definition = int(eds_record.default_value).to_bytes(4, "big")
```

Comment:
Is there a reason for switching the base value from 16 to 10 here?

The DCF's should record value in hex, if I'm not mistaken.

Reply:
Yes, when the base is specified as 16, a `TypeError` shows up in CM that says `int() can't convert non-string with explicit base`; defaulting to base 10 alleviates the error and does not seem to parse anything incorrectly.